src/traits.c

```diff
@@ -385,7 +385,7 @@ Expression *pointerBitmap(TraitsExp *e)
             d_uns64 arrayoff = offset;
             d_uns64 nextsize = t->next->size();
             d_uns64 dim = t->dim->toInteger();
-            for (size_t i = 0; i < dim; i++)
+            for (size_t i = 0; i < (size_t)dim; i++)
```

Comment:
If `i` is used to point dmd internal data, limiting it to C `size_t` is legitimate. but in here, `i` is used to calculate `offset`, and `offset` should be limited by the D `size_t`.
I think current code may cause problem when 32bit host dmd compiles 64bit target code.